web/client/epics/login.js

```diff
@@ -42,6 +43,9 @@ const refreshTokenEpic = (action$, store) =>
 const reloadMapConfig = (action$, store) =>
     action$.ofType(LOGIN_SUCCESS, LOGOUT)
     .filter(() => pathnameSelector(store.getState()).indexOf("viewer") !== -1)
+    .filter((data) => {
+        return data.type !== "LOGOUT" ? hasMapAccessLoadingError(store.getState) : true;
```

Comment:
store.getState is a function, I'm afraid `hasMapAccessLoadingError(store.getState) will always return false